src/export/Export.cpp

```diff
@@ -249,19 +249,24 @@ std::unique_ptr<Mixer> ExportPlugin::CreateMixer(const TrackList &tracks,
                   true, mixerSpec);
 }
 
-void ExportPlugin::InitProgress(std::unique_ptr<ProgressDialog> &pDialog,
+void ExportPlugin::InitProgress(std::unique_ptr<BasicUI::ProgressDialog> &pDialog,
    const TranslatableString &title, const TranslatableString &message)
 {
    if (!pDialog)
       pDialog = std::make_unique<ProgressDialog>( title, message );
-   else {
-      pDialog->SetTitle( title );
-      pDialog->SetMessage( message );
-      pDialog->Reinit();
+   else
+   {
+      if (auto pd = dynamic_cast<ProgressDialog*>(pDialog.get()))
```

Comment:
Rather than this `dynamic_cast`, the abstract class `BasicUI::ProgressDialog` can have two more virtual functions.  `SetTitle` has an obvious purpose.  `Reinit` should be commented as resetting progress to zero and resetting any cancelled or stopped state.

Reply:
`SetTitle` comes from wxWidgets. I can call it SetDialogTitle though